AlphaWallet/Common/Views/TextView.swift

```diff
@@ -83,7 +83,7 @@ class TextView: UIControl {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
         label.font = Configuration.Font.textFieldTitle
-        label.textColor = Configuration.Color.Semantic.label
```

Comment:
Maybe `Configuration.Color.Semantic.label` can be removed too?

Reply:
But it builds after I remove the line `static let label = Colors.appGrayLabel`. Where else is it used?

Reply:
> How do I find screens for subtitleColor/TokenCardRowViewModelProtocol and subtitleColor/OpenSeaNonFungibleTokenCardRowViewModel?

Is it relevant here? Otherwise, can you ask this elswhere? Harder to track

Reply:
Ok I also removed .secondary which affected the two screens mentioned about. Removing .label alone didn't cause any issues.